lib/agent/actions/fileretrieval/index.js

```diff
@@ -17,55 +19,84 @@ var fs = require('fs'),
 var common = require('./../../common'),
     system = common.system,
     node_bin = path.join(system.paths.current, 'bin', 'node'),
-    run_as_user = common.system.run_as_user,
     logger = common.logger,
     os_name = process.platform.replace('darwin', 'mac').replace('win32', 'windows');
 
 var em,
     cp;
 
+var UPLOAD_SERVER = 'http://solid.preyproject.com/upload/upload';
+
 exports.check_pending_files = function() {
   files.run_stored();
 }
 
 exports.start = function(options, cb) {
   var file_path = options.path,
-      user = options.user,
-      file_id = options.file_id;
+      file_id = options.file_id,
+      file_size = parseInt(options.size),
```

Comment:
value should already be a int, so callers of filetrieval.start should provide a numeric value for file_size